app/config.go

```diff
@@ -0,0 +1,13 @@
+package app
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func init() {
```

Comment:
[for reviewers] `init()` is called once regardless of how many times this package is imported per https://tutorialedge.net/golang/the-go-init-function/

IMO this seems less clear than an explicit `New()` or `NewConfig()` invocation but maybe that's because I haven't seen it before. If it's clearer for celestia-node then :shipit: